src/components/dashboard.js

```diff
@@ -91,27 +78,30 @@ const useStyles = makeStyles((theme) => ({
       width: theme.spacing(9),
     },
   },
-  appBarSpacer: theme.mixins.toolbar,
-  content: {
-    flexGrow: 1,
-    height: '100vh',
-    overflow: 'auto',
-  },
-  container: {
-    paddingTop: theme.spacing(4),
-    paddingBottom: theme.spacing(4),
-  },
-  paper: {
-    padding: theme.spacing(2),
-    display: 'flex',
-    overflow: 'auto',
-    flexDirection: 'column',
-  },
-  fixedHeight: {
-    height: 240,
-  },
 }));
 
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      light: '#758afe',
+      main: '#536dfe',
+      dark: '#3a4cb1',
+      contrastText: '#ffffff',
+    },
+    secondary: {
+      light: '#ef6694',
+      main: '#ec407a',
+      dark: '#a52c55',
+      contrastText: '#ffffff',
+    },
+  },
+});
```

Comment:
Put this on separate file on `src/theme/theme.js`. We'll just do this on new PR.

Reply:
Ok.